osgp/platform/osgp-core/src/main/resources/db/migration/V20200914092100000__Adds_domain_info_column_to_rtu_devices_table.sql

```diff
@@ -0,0 +1,15 @@
+DO
+$$
+BEGIN
+
+IF NOT EXISTS (
+  SELECT
+    *
```

Comment:
Please avoid using `SELECT *`. There is no need to extract all information for this SQL migration script. Have a look at this example: https://github.com/OSGP/open-smart-grid-platform/blob/development/osgp/platform/osgp-core/src/main/resources/db/migration/V20180917095300666__Adds_column_to_protocol_info.sql